test/cypress/integration/widgets/filter_spec.ts

```diff
@@ -8,36 +8,59 @@ describe('Filter', () => {
     });
 
     it('fills dropdowns with correct data', () => {
+        const getDropdownItems = (id?: string) => {
+            if (id) {
+                return cy.get(`${id} .menu > *`);
+            }
+            return cy.get('.menu > *');
+        };
```

Comment:
Optional:

Will shorter version:
```suggestion
        const getDropdownItems = (id?: string) => id ? cy.get(`${id} .menu > *`) : cy.get('.menu > *');
```
work here?

Reply:
changed as you wish